disperser/dataapi/v2/batches.go

```diff
@@ -194,18 +194,28 @@ func (s *ServerV2) FetchBatch(c *gin.Context) {
 		errorResponse(c, errors.New("invalid batch header hash"))
 		return
 	}
-	batchHeader, attestation, err := s.blobMetadataStore.GetSignedBatch(c.Request.Context(), batchHeaderHash)
-	if err != nil {
-		s.metrics.IncrementFailedRequestNum("FetchBatch")
-		errorResponse(c, err)
-		return
+	signedBatch, ok := s.signedBatchCache.Get(batchHeaderHashHex)
+	if !ok {
```

Comment:
I would rename `ok`->`cached` here and elsewhere. It makes the cache check easier to read `if !cached {`

Reply:
Sg